core/src/main/java/com/google/zxing/oned/Code128Reader.java

```diff
@@ -238,6 +239,8 @@ public Result decodeRow(int rowNumber, BitArray row, Map<DecodeHintType,?> hints
 
     boolean convertFNC1 = hints != null && hints.containsKey(DecodeHintType.ASSUME_GS1);
 
+    String symbologyModifier = "0";
```

Comment:
Should this just be an int?

Reply:
Aggreed for code consistency. Technical no difference, it's concatenated to a String anyway.